### ISSUE TITLE
Add Support for magnet: links

### ISSUE THREAD

**Issue by saltedlolly** [bulljit/Transmission-Add-Torrent-Bookmarkelet#9]
I'm not sure if Transmission can support this via RPC but it could be a nice addition.

**Comment by bulljit** [comment]
Can you check to see what happens when you add a magnet link via the WebUI?

Alright I gotta head off to my morning commute, ttyl

**Comment by saltedlolly** [comment]
Just tested it and to my great surprise it worked. I'm sure last time I tried it didn't but that may have been a bandwidth issue. :)

**Comment by saltedlolly** [comment]
The only slight issue with it is a cosmetic one. The success message inserts '+' symbols where spaces should be. e.g.:

Torrent+Name+2010+720p

Also the word wrapping issue is still there, with long names going outside the box. 

They're small cosmetic issues I know, but it bugs me! ;)

**Comment by bulljit** [comment]
Dang it, i added more CSS line to fix that word-wrap last night, are you using the cookie-string bookmarklet?  And updated client which contains an updated fetchtorrent.html?

As far as the name goes, that's a reply from the RPC, I'd rather not change it as it will probably break some other torrent names :)

If my new code doesn't fix the word-break issue, it's your turn to google it :) Browsers are way too finicky with CSS.

**Comment by saltedlolly** [comment]
I fixed the text wrapping issue. It took my ages of trial an error using Firebug but it worked in the end:

`body {`

`background-color: white;`

`font-family: Helvetica,sans-serif;`

`font-size: 16px;`

`margin: 0;`

`padding: 0;`

`width: 320px;`      <-- Added width.

`}`

`#diag_list {`

`background: url("images/graphics/bookmarklet/diag_mid.png") repeat-y scroll 0% 0% transparent;`

`color: #333333;`

`font-family: Arial,Helvetica,sans-serif;`

`font-size: 16px;`    <-- For some reason the font needs to be bigger

`font-weight: bold;`

`max-width: 0px;`     <--  I have no idea why this fixes it! why zero?

`padding-left: 10px;`

`padding-right: 10px;`

`text-align: center;`       <---  Centered things to make it pretty. :)

`word-wrap: break-word;`

`}`

Don't ask me why it works, but it does!

Update: Shit - There seem to still be some more issues. I think we made need to rethink the page layout. This is beyond me!
